modServer.bas

```diff
@@ -3481,85 +3481,6 @@ Function NoDirectionalWalls(TheMap As Long, X As Long, Y As Long, Direction As L
     End Select
 End Function
 
-Function NoDirectionalWalls2(TheMap As Long, X As Long, Y As Long, Direction As Long) As Boolean

```

Comment:
I just checked [Remote's original source](https://github.com/TheRemote/OdysseyClassic/blob/master/Server/modServer.bas) and these lines are included in there. So it's just random buggy code with no apparent use? How strange.